jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -1198,6 +1199,11 @@ YUI.add('juju-env-api', function(Y) {
         'num-units': numUnits
       };
 
+      if (series) {
+        // We only want to supply the series when it is a multi-series charm.
```

Comment:
When this was originally written I had a different architecture to get to this point. That's been fixed because it was bad :) I'll update this comment because now the only issue is that if series is defined but `undefined` (somehow) then the deploy doesn't complete successfully.